src/overlays/actors/ovl_En_Skb/z_en_skb.h

```diff
@@ -13,6 +13,24 @@ typedef enum {
     /* 0 */ ENSKB_PARAM_0
 } EnSkbParam;
 
+typedef enum EnSkbBodyPart {
+    /*  0 */ ENSKB_BODYPART_0,
+    /*  1 */ ENSKB_BODYPART_1,
```

Comment:
This was mentioned in a previous review, but the `ENSKB_` vs `EN_SKB_` vs `SKB_` is widely inconsistent. I mostly just tried to follow what was already done for that actor instead of consistency